### PR TITLE
NXP-25380: use JWT auth

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

File: src/test/java/org/nuxeo/wopi/jaxrs/TestFilesEndpoint.java [hunk 1]
```diff
@@ -73,15 +74,16 @@
...
 import org.nuxeo.ecm.platform.test.PlatformFeature;
 import org.nuxeo.ecm.platform.usermanager.UserManager;
 import org.nuxeo.ecm.restapi.test.RestServerFeature;
-import org.nuxeo.ecm.tokenauth.service.TokenAuthenticationService;
 import org.nuxeo.jaxrs.test.CloseableClientResponse;
 import org.nuxeo.jaxrs.test.JerseyClientHelper;
 import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.LocalDeploy;
```
Comment (efge) [inline src/test/java/org/nuxeo/wopi/jaxrs/TestFilesEndpoint.java]: Unused import.

**Comment by efge** [comment]
👍

**Comment by kevinleturc** [comment]
👍

**Comment by efge** [comment]
👍

**Pull request merged**